.github/workflows/selfdrive_tests.yaml

```diff
@@ -51,7 +51,7 @@ jobs:
       timeout-minutes: ${{ ((steps.restore-scons-cache.outputs.cache-hit == 'true') && 10 || 30) }} # allow more time when we missed the scons cache
       run: |
         cd $STRIPPED_DIR
-        ${{ env.RUN }} "python selfdrive/manager/build.py"
+        ${{ env.RUN }} "CI=1 python selfdrive/manager/build.py"
```

Comment:
looks like this was a conflict that wasn't resolved properly? removed that here: https://github.com/commaai/openpilot/commit/86bd3379522c67da94a3ac6f3e0628bffa14ce50